docs/virtualkeyboard/index.html

```diff
@@ -0,0 +1,492 @@
+                                            height: 100vh;
+                                            grid-template-columns: env(fold-left) calc(100vw - env(fold-right));
+                                            grid-column-gap: calc(env(fold-right) - env(fold-left));
+                                        }
+                                        .map {
+                                            grid-column: 1;
+                                            grid-row: 1;
+                                        }
+                                        .search-box {
+                                        /* overlay into same grid column as the map */
+                                            grid-column: 1;
+                                            grid-row: 1;
+            
+                                            /* position the search box at the bottom of the cell, centered */
+                                            align-self: end;
+            
+                                            /* margin when keyboard not showing on left screen */
+                                            margin: 20px;
+            
+                                            /* give it a little style */
+                                            padding: 10px;
+                                            font-size: 24px;
+                                            border: 4px solid black;
+                                            border-radius: 4px;
+                                            background-color: #86DBF6;
+                                        }
+                                        .locations-list {
+                                            grid-column: 2;
+                                            grid-row: 1;
+                                        }
+            
+                                        @media (env(keyboard-inset-right) <= env(fold-left)) {
+                                        /* keyboard is on the left screen, adjust search box */
+                                            .search-box {
+                                                margin-bottom: calc(20px + env(keyboard-inset-height));
+                                            }
+                                        }
+                                    }
+                                </style>
+                                <div class="map">...</div>
+                                <input class="search-box" type="search" placeholder="search...">
+                                <div class="locations-list">...</div>
+                                <script type="module">
+                                    navigator.virtualKeyboard.overlaysContent = true
+                                </script>
+                            </xmp>
+                        </pre>
+                        </aside>
+            </section>
+    </section>
+    <section>
+        <h2>Privacy and Security Considerations</h2>
```

Comment:
Several pieces of information in this section seem like a set of claims that this API doesn't come with significant risk to privacy or security.  I don't think we need to cover that.  I expected this section to enumerate important implementation details that UAs may or must follow to ensure a secure and privacy preserving implementation.  Additionally, I think it would be appropriate to enumerate the responsibilities of authors, if any, to ensure they don't create privacy or security issues when using the API.  

Maybe [IndexedDB](https://www.w3.org/TR/IndexedDB-2/#privacy) is a good example.

Reply:
Updated. Is this what you had in mind?